website/docs/docs/deploy/continuous-integration.md

```diff
@@ -27,6 +27,27 @@ When the CI run completes, you can view the run status directly from within the
 
 dbt Cloud deletes the temporary schema from your <Term id="data-warehouse" /> when you close or merge the pull request. If your project has schema customization using the [generate_schema_name](/docs/build/custom-schemas#how-does-dbt-generate-a-models-schema-name) macro, dbt Cloud might not drop the temporary schema from your data warehouse. For more information, refer to [Troubleshooting](/docs/deploy/ci-jobs#troubleshooting).
 
+## Git providers who support CI
+
+If your git provider has a native dbt Cloud integration, you can set up continuous integration (CI) jobs within dbt Cloud.
+
+| **Git provider** | **Native dbt Cloud integration** |[**Automated CI job**](/docs/deploy/ci-jobs)|**Git import with URL**| **Information**|
+| -----------------| ---------------------------------| -------------------------------------------|-----------------------|---------|
+|[Azure DevOps](/docs/cloud/git/setup-azure) <Lifecycle status="enterprise" />|  ✅|  ✅ |  ✅  | Organizations on the Team Plan can connect to Azure DeveOps via deploy key. Note, you won’t get the automated CI jobs but you can still develop.|
+|[GitHub](/docs/cloud/git/connect-github) <Lifecycle status="developer,team,enterprise" /> | ✅ | ✅ |  ✅         |                                  
+|[GitLab](/docs/cloud/git/connect-gitlab) <Lifecycle status="developer,team,enterprise" /> | ✅ | ✅ |  ✅         |
+|All other git providers imported with a [git URL](/docs/cloud/git/import-a-project-by-git-url) ([BitBucket](/docs/cloud/git/import-a-project-by-git-url#bitbucket), [AWS CodeCommit](/docs/cloud/git/import-a-project-by-git-url#aws-codecommit), and others)| ❌    | ❌    | ✅   | Bitbucket doesn’t have a native dbt Cloud integration, refer to the [Customizing CI/CD with custom pipelines](/guides/custom-cicd-pipelines?step=1) guide to set up continuous integration and continuous deployment (CI/CD).|
+
+
+### No native integration
```

Comment:
not sure i understand why this section was turned into a snippet? and the snippet is only used once? snippets are usually for multiple uses in diff pages?

Reply:
also, the content is teh same as this here: https://docs.getdbt.com/docs/deploy/ci-jobs#trigger-a-ci-job-with-the-api so wondering if we need thi 'no native' section?

Reply:
![Screenshot 2024-12-19 at 14 42 22](https://github.com/user-attachments/assets/ed644d55-f721-40bd-b264-c602ffe1fd64)